tests/e2e/framework/istioctl.go

```diff
@@ -118,6 +119,27 @@ func (i *Istioctl) run(args string) error {
 	return nil
 }
 
+// RunWithOutput runs istioctl with the named args, optionally overriding the namespace with `ns` and the
+// istio namespace with `istioNs`.  `expectFailure` suppresses logging of failure when the command is
+// expected/desired to fail by the test.
+func (i *Istioctl) RunWithOutput(args []string, ns string, istioNs string, expectFailure bool) (output string, err error) {
+	if ns == "" {
+		ns = i.namespace
+	}
+	if istioNs == "" {
+		istioNs = i.istioNamespace
+	}
+
+	if output, err = util.Shell(fmt.Sprintf("%s --namespace %s --istioNamespace %s %s",
+		i.binaryPath, ns, istioNs, strings.Join(args, " "))); err != nil {
+		if !expectFailure {
```

Comment:
True.  I put it here because I needed i.binaryPath, i.namespace, and i.istioNamespace.  I tried to make this a generic method to call any istioctl command.  For example (although I haven't done it, I considered writing a test to make sure `istioctl foo` doesn't panic.)  Such a test couldn't be written without a general library function to execute the downloaded _istioctl_.